plugins/deprecate_bot_prefixes.py

```diff
@@ -0,0 +1,21 @@
+from errbot import BotPlugin
+import config
+
+class DeprecateBotPrefixes(BotPlugin):
+    """
+    A callback for every message that starts with depecrated prefix, hence, leaving a deprecation notice
+    """
+
+    def callback_message(self, msg):
+        """
+        Notify the user issuing the command that use deprecated prefix.
+        """
+
+        for deprecated_prefix in config.BOT_DEPRECATED_PREFIXES:
```

Comment:
use `self.bot_config.BOT_DEPRECATED_PREFIXES` instead of `config.BOT_DEPRECATED_PREFIXES`